### ISSUE TITLE
Homebrew formula - dependencies, etc. - getting it done

### ISSUE THREAD

**Issue by ylluminate** [hteso/iaito#210]
So I thought perhaps we could get a conversation going about getting a Homebrew formula going for macOS as per #64. 

It seems to me that the first question to ask is: why are we on QT 5.6?  Right now the main stable is at `5.9.1` for brew, but also there are `5.5.1` and `5.7.1` aliases.  Do we need to make a `5.6.x` alias or do can we move `iaito` forward to 5.7?

**Comment by xarkes** [comment]
For the Qt version questiok, I think we should move iaito to the latest Qt version

**Comment by ylluminate** [comment]
What's preventing it from being migrated to the most recent QT? Has anyone tried it yet and hit issues?

**Comment by xarkes** [comment]
Nothing I'd say, just not enough time for now.

**Comment by xarkes** [comment]
Actually I think iaito is compatible with the latest Qt version (5.9.1)

**Comment by xarkes** [comment]
Confirmed.

**Comment by nucleare2** [comment]
Yes I can confirm as well so this should not have been closed. The original intent of this ticket was to get a Homebrew recipe together, not the QT issue. So let's talk about what's next @xarkes.

**Comment by xarkes** [comment]
Ah sorry I read too fast. Then open a PR with the recipe

**Comment by nucleare2** [comment]
Sounds good. Let's see if we can coordinate a recipe here. I think the QT question was probably the biggest potential obstacle, the rest is probably pretty straight forward.

**Comment by ylluminate** [comment]
Uhm, I've been looking into it here, but I don't think it's possible in that the Homebrew guys seem [unwilling to accept head-only formulae](https://devhub.io/repos/Homebrew-homebrew-head-only). @xarkes can we get a release tag at least for this even if it is a beta or alpha tag? If we can get a version that has a zip file that can be pulled down I think we'll be fine.

**Comment by Maijin** [comment]
See https://github.com/hteso/iaito/pull/170

**Comment by ylluminate** [comment]
@Maijin what is causing the #170 to be stalled? Originally I had hoped to push the issue (#64) forward by getting this going. Seems that #64 & #170 are duplicates that are not getting attention. Does the logic not hold true that if we can get this formula together, even as head-only, it will push those two issues forward through resolution?  In other words, won't a Homebrew recipe that reliably builds all of the time with the most recent QT, etc. get us to where Travis will be able to leverage this to give us what we want for continual builds?

**Comment by Maijin** [comment]
Read the issue, it needs @hteso to provide with tokens for travis

**Comment by ylluminate** [comment]
Is there anything preventing @hteso from doing this or do we need to set up a new build server if he's not available?
